etstool.py

```diff
@@ -342,6 +342,7 @@ def changelog(ctx):
             "removal": "Removals",
             "doc": "Documentation changes",
             "test": "Test suite",
+            "build": "Build System",
```

Comment:
Note that the ordering here is used for the changelog too. The ordering should be more or less consistent with the ordering seen in Traits or TraitsUI's changelogs.